db/knex_migrations/2025-06-14-0000-heartbeat-range-config.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+    return knex.schema.alterTable("status_page", function (table) {
+        table.integer("heartbeat_bar_days").notNullable().defaultTo(0);
```

Comment:
I think this is what we want.

```suggestion
        table.smallint("heartbeat_bar_days").notNullable().defaultTo(0).checkBetween([0, 365]);
```